app-text/odt2txt/odt2txt-0.5.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="A simple converter from OpenDocument Text to plain text"
+DESCRIPTION="odt2txt is a command-line tool which extracts the text out of \
+OpenDocument Texts produced by LibreOffice, OpenOffice, StarOfficem, KOffice \
+and others."
+HOMEPAGE="https://github.com/dstosberg/odt2txt/"
+COPYRIGHT="2002-2016 Dennis Stosberg"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/dstosberg/odt2txt/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="23a889109ca9087a719c638758f14cc3b867a5dcf30a6c90bf6a0985073556dd"
+PATCHES="odt2txt-0.5-makefile-fix.patch"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	odt2txt = $portVersion
+	cmd:odt2txt = $portVersion
+	"
+REQUIRES="
+	haiku
+	lib:libz
+	lib:libiconv
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	devel:libz
+	devel:libiconv
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc
+	cmd:groff
+	cmd:make
+	"
+
+BUILD()
+{
+	make $jobArgs
+}
+
+INSTALL()
+{
+	# Upstream Makefile uses $(DESTDIR)$(BINDIR) and $(DESTDIR)$(MAN1DIR),
+	# hence the need to have a trailing slash in DESTDIR.
+	make install DESTDIR="$prefix/" BINDIR="$relativeBinDir" \
+	             MANDIR="$relativeManDir"
```

Comment:
The error comes from the Makefile. It defines:
```
DESTDIR = /usr/local
PREFIX  =
```
instead of:
```
DESTDIR = 
PREFIX  = /usr/local
```
which would be much better.

BTW, using
```
	make install DESTDIR= PREFIX=$prefix MANDIR=$manDir
```
allows to avoid having to specify BINDIR and does not need any trailing slash either ;-)